File: engine/model/src/main/scala/tools/packet/guild/UpdateGuildPoints.scala
package tools.packet.guild

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class UpdateGuildPoints(private var _guildId: Int, private var _points: Int) extends PacketInput {
  def guildId: Int = _guildId

  def points: Int = _points

  override def opcode(): SendOpcode = SendOpcode.GUILD_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/guild/UpdateGuildPoints.java
package tools.packet.guild;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record UpdateGuildPoints(Integer guildId, Integer points) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.GUILD_OPERATION;
   }
}